pkg/registry/generic/etcd/etcd_test.go

```diff
@@ -97,13 +97,21 @@ func (sm SetMatcher) Matches(obj runtime.Object) (bool, error) {
 	return sm.Has(pod.Name), nil
 }
 
-// EverythingMatcher matches everything
-type EverythingMatcher struct{}
+func (setMatcher) MatchesSingle() (string, bool) {
+	return "", false
```

Comment:
Oh, I think you're right-- it matches by name which is indeed its key. Good catch.